rox_bringup/launch/bringup_sim_launch.py

```diff
@@ -46,7 +51,7 @@ def execution_stage(context: LaunchContext, frame_type, rox_type, arm_type, use_
         PythonLaunchDescriptionSource(
             os.path.join(get_package_share_directory('ros_gz_sim'), 'launch', 'gz_sim.launch.py')
         )
-        , launch_arguments={'ign_args': ['-r ', default_world_path]}.items()
+        , launch_arguments={'gz_args': ['-r ', default_world_path]}.items()
```

Comment:
could you also change all the other ign references to gz ?

Reply:
this seems to be the only one anna